coderd/workspaceagents_test.go

```diff
@@ -1210,6 +1210,9 @@ func TestWorkspaceAgent_Metadata(t *testing.T) {
 	require.NoError(t, err)
 
 	// Verify manifest API response.
+	//  community: allow api users to identify workspace_id & owner by agent tokens
```

Comment:
Let's move this comment inline in the agentsdk definition.

Reply:
done